src/main/osd/osd_elements.c

```diff
@@ -1293,7 +1333,10 @@ const osdElementDrawFn osdElementDrawFunction[OSD_ITEM_COUNT] = {
     [OSD_COMPASS_BAR]             = osdElementCompassBar,
 #ifdef USE_ESC_SENSOR
     [OSD_ESC_TMP]                 = osdElementEscTemperature,
+#endif
+#if defined(USE_RPM_FILTER) || defined(USE_ESC_SENSOR)
     [OSD_ESC_RPM]                 = osdElementEscRpm,
+    [OSD_ESC_RPM_FREQ]            = osdElementEscRpmFreq,
```

Comment:
We would prefer the ordering stays synchronized with `osd_items_e` to make it easier to ensure that no elements were missed.

Reply:
It is fixed.